wavesrv/pkg/cmdrunner/cmdrunner.go

```diff
@@ -1125,7 +1130,10 @@ func prettyPrintByteSize(size int64) string {
 }
 
 // this can only be called in a defer func, because recover() only works inside of a defe
-func deferWriteCmdStatus(ctx context.Context, cmd *sstore.CmdType, startTime time.Time, exitSuccess bool, outputPos int64) {
+func deferWriteCmdStatus(ctx context.Context, cmd *sstore.CmdType, startTime time.Time, exitSuccess bool, outputPos int64, outputPty bool) {
```

Comment:
if outputPty is true, no need to close the cmd, so this function just returns

I'm not sure if that is clear to someone who is reading this for the first time. I maybe should change the name of this variable or add a comment